### PR TITLE
Get, GetConfig, GetOper, EditConfig, StartSession, CloseSession, KillSession

### PR DESCRIPTION
**Pull request by remingtonc**
Method support for multiple RPCs. Decided to get a little wild and follow the [numpy docs](https://numpydoc.readthedocs.io/en/latest/format.html#numpydoc-docstring-guide) (imprecisely) yielding a lot of doc..

Just realized does not yet have `setup.py`, will commit later.

Made a potentially bad decision centralizing the gRPC method control flow into `Client.__fulfill_request` ([code](https://github.com/cisco-grpc-connection-libs/nx-os-grpc-python/blob/development/nxos_grpc/client.py#L201)), not sure how to feel about it yet but for now everything works the same way across all RPCs.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+verify_ssl = true
+name = "pypi"
+
+[dev-packages]
+grpcio-tools = "*"
+googleapis-common-protos = "*"
+pylint = "*"
+
+[packages]
+grpcio = "*"
```
Comment (skkumaravel) [inline Pipfile]: Is this really * or 1.0.0 like XR?

**Review by skkumaravel — COMMENTED** [review]
Do you want gRPC Dialout to be part of this?

File: Pipfile (hunk 1 above)
Comment (remingtonc) [reply Pipfile]: Good question..it's working with..
```
➜ pip list
Package    Version
---------- -------
enum34     1.1.6
grpcio     1.13.0
pip        10.0.1
protobuf   3.6.0
setuptools 39.2.0
six        1.11.0
wheel      0.31.1
```
We should ask engineering

**Comment by remingtonc** [comment]
@skkumaravel Receiving dialout would be cool - do you have any insights on that? Let's do a separate pull request there.

**Comment by remingtonc** [comment]
@skkumaravel gRPC implementation is based around 0.11.1 which is pre-2016 and will be a rabbit hole to build around. Keeping it latest until we have an issue.

**Review by ziausyed — APPROVED** [review]

**Review by skkumaravel — APPROVED** [review]

**Pull request merged**